src/Console/Command/CompareCommand.php

```diff
@@ -0,0 +1,323 @@
+
+    /** @var int $countInherited */
+    private $countInherited = 0;
+
+    /** @var array $ruleSets Stores all the RuleSets and the Fixers they have */
+    private $ruleSets = [];
+
+    /** @var array $list */
+    private $list = [];
+
+    /** @var array $dump */
+    private $dump;
+
+    /**
+     * @param null|FixerFactory $fixerFactory
+     */
+    public function __construct(FixerFactory $fixerFactory = null)
+    {
+        parent::__construct();
+
+        if (null === $fixerFactory) {
+            $fixerFactory = new FixerFactory();
+            $fixerFactory->registerBuiltInFixers();
+        }
+
+        $this->fixerFactory = $fixerFactory;
+        $this->defaultConfig = new Config();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName(self::COMMAND_NAME)
+            ->setDefinition(
+                [
+                    new InputOption('config', '', InputOption::VALUE_REQUIRED, 'The path to a .php_cs file.'),
+                    new InputOption('hide-configured', '', InputOption::VALUE_NONE, 'Hide fixers that are configured (in the config file or because of inheritance).'),
+                    new InputOption('hide-enabled', '', InputOption::VALUE_NONE, 'Hide fixers that are currently enabled (the ones that are not disabled with [\'fixer_name\' => false]).'),
+                    new InputOption('hide-risky', '', InputOption::VALUE_NONE, 'Hide fixers that are marked as risky.'),
+                    new InputOption('hide-inherited', '', InputOption::VALUE_NONE, 'Hide fixers that inherited from RuleSets.'),
+                    new InputOption('dump', '', InputOption::VALUE_NONE, 'Dumps the comparing result in a copy-and-pastable format ready for the .php_cs file.'),
+                ]
+            )
+            ->setDescription('Compares existent Fixers with the ones actually configured or enabled by inheritance.')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->initialize($input, $output);
+
+        /**
+         * @var string
+         * @var RuleSet $set
+         */
+        foreach ($this->ruleSets as $name => $set) {
+            /**
+             * @var string
+             * @var bool   $value
+             */
+            foreach ($set->getRules() as $rule => $value) {
+                if (false === isset($this->list[$rule]['in_set'])) {
+                    $this->list[$rule]['in_set'] = [];
+                }
+                $this->list[$rule]['in_set'][] = $name;
+            }
+        }
+
+        /** @var FixerInterface $fixer */
+        foreach ($this->builtInFixers as $fixer) {
+            if ($isInherited = !empty($this->list[$fixer->getName()]['in_set'])) {
+                ++$this->countInherited;
+            }
+
+            $this->list[$fixer->getName()]['name'] = $fixer->getName();
```

Comment:
Yes, maybe: I like the suggestion... I called the method `processFixer` (because it actually doesn't simply create a row, but makes the checks to get the info about the `Fixer`) and added also another method `processRuleSet`: the `execute()` method is now cleaner.